web-ui/src/helpers/query-parameters.js

```diff
@@ -34,13 +47,21 @@ export const useQueryParameters = qps => {
         qp.setter(v || qp.default);
       }
     }
-  }, []);
+    if (processedQPs) processedQPs.current = true;
```

Comment:
If the caller supplied this ref, we need to set its value to to `true` to include that we have processed the query parameters. This prevents use from doing it again later.